svelte-cloudinary/src/lib/components/CldImage.spec.js

```diff
@@ -6,7 +6,7 @@ import CldImage from './CldImage.svelte';
 
 describe('CldImage', () => {
     it('should render a Cloudinary image with the given attributes', () => {
-        const expectedUrl = 'image/upload/c_limit,w_300/b_#123456/f_auto/q_auto/sample?_a=BBECcRAD0';
```

Comment:
@matiasfha I think we might need to remove this param in the tests, because when I run the tests locally I get a different value (I'm assuming it's a cache key, or something like that)?

Removing allows the tests to pass.